.rubocop.yml

```diff
@@ -0,0 +1,18 @@
+require:
+  - rubocop-rails
+  - rubocop-rspec
+  - rubocop-performance
+
+AllCops:
+  NewCops: enable
+  Exclude:
+    - 'Gemfile'
+    - 'bin/**/*'
+    - 'db/**/*'
+    - 'config/**/*'
+    - 'vendor/**/*'
+    - 'spec/*_helper.rb'
```

Comment:
I'm just curious, why do we ignore `*_helper.rb` and `bin`?

Reply:
Great question! I don't know the actual answer, but in this case:

- I installed rubocop and it reformatted everything in `/bin` and the helper files
- I went "huh...that's a lot of changes that it would be annoying to review in the PR"
- I saw that we ignore them in other repos

...and that's why I did it. 🤷🏻